tests/infra/basicperf.py

```diff
@@ -258,6 +268,8 @@ def run(args):
                     os.path.abspath(path_to_requests_file),
                     "--pid-file-path",
                     "cmd.pid",
+                    "--boot-time",
+                    str(boot_time_us),
```

Comment:
I'm not sure why we pass this in rather than offset the agg dataframe in one go, but this is fine too.